sql-gremlin/src/main/java/org/twilmes/sql/gremlin/adapter/util/SqlGremlinError.java

```diff
@@ -0,0 +1,105 @@
+package org.twilmes.sql.gremlin.adapter.util;
```

Comment:
Please add license headers.
Please also add a checkstyle rule that enforces presence of these headers.